ui/src/app/machines/views/MachineDetails/MachineConfiguration/MachineForm/MachineForm.tsx

```diff
@@ -84,7 +84,7 @@ const MachineForm = ({ systemId }: Props): JSX.Element | null => {
           buttons={FormCardButtons}
           cleanup={cleanup}
           editable={editing}
-          errors={errors}
+          errors={editing ? errors : undefined}
```

Comment:
Does this need to be `undefined` here? If so maybe leave a comment (or maybe see if it can be fixed so it doesn't have to be `undefined`) otherwise this should probably use `null`.

Reply:
Changed to `null`